saleor/dashboard/category/views.py

```diff
@@ -22,7 +22,14 @@ def category_list(request):
     categories = get_paginator_items(
         category_filter.qs, settings.DASHBOARD_PAGINATE_BY,
         request.GET.get('page'))
-    ctx = {'categories': categories, 'filter': category_filter}
+    summary_msg = npgettext_lazy(
+        'Filter set results summary',
+        'Found %(counter)d matching category',
+        'Found %(counter)d matching categories',
+        'counter') % {'counter': category_filter.qs.count()}
```

Comment:
What is the word `counter` doing here as the `number` parameter?

Reply:
Which brings me to another question, why is it a lazy call? 😄

Reply:
Changed it to simple ```npgettext```.